js_modules/dagit/packages/core/src/assets/AssetView.tsx

```diff
@@ -458,39 +453,44 @@ export const ASSET_VIEW_DEFINITION_QUERY = gql`
   ${UNDERLYING_OPS_ASSET_NODE_FRAGMENT}
 `;
 
-const HistoricalViewAlert: React.FC<{
+const HistoricalViewAlert = ({
+  asOf,
+  hasDefinition,
+}: {
   asOf: string | undefined;
-  onClick: () => void;
   hasDefinition: boolean;
-}> = ({asOf, onClick, hasDefinition}) => (
-  <Box
-    padding={{vertical: 16, horizontal: 24}}
-    border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
-  >
-    <Alert
-      intent="info"
-      title={
-        <span>
-          This is a historical view of materializations as of{' '}
-          <span style={{fontWeight: 600}}>
-            <Timestamp
-              timestamp={{ms: Number(asOf)}}
-              timeFormat={{showSeconds: true, showTimezone: true}}
-            />
+}) => {
+  const {pathname} = useLocation();
+  return (
+    <Box
+      padding={{vertical: 16, horizontal: 24}}
+      border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
+    >
+      <Alert
+        intent="info"
+        title={
+          <span>
+            This is a historical view of materializations as of{' '}
+            <span style={{fontWeight: 600}}>
+              <Timestamp
+                timestamp={{ms: Number(asOf)}}
+                timeFormat={{showSeconds: true, showTimezone: true}}
+              />
+            </span>
+            .
           </span>
-          .
-        </span>
-      }
-      description={
-        <ButtonLink onClick={onClick} underline="always">
-          {hasDefinition
-            ? 'Show definition and latest materializations'
-            : 'Show latest materializations'}
-        </ButtonLink>
-      }
-    />
-  </Box>
-);
+        }
+        description={
+          <Link to={`${pathname}?time=${asOf}`}>
```

Comment:
Ahh thanks for jumping on this! You beat me to it 🙌  

I think this will work but this alert appears outside the tab area of the AssetView and I'm worried that making this just pathname + time could strip the `?view=partitions` param and jump you to the events tab from the partitions tab. 

It's kind of a pain but maybe this to update just the one param?

``` 
return `${pathname}?${new URLSearchParams({...Object.fromEntries(new URLSearchParams(search)), asOf}).toString()}`
```

Reply:
Oh, yeah, I wasn't sure if it was necessary to preserve those. Will update.